src/StatsTable.tsx

```diff
@@ -99,11 +99,13 @@ export function StatsTable({ pokemon }: StatsTableProps) {
         aria-label={hasRoomForFullText ? "" : t("pokedex.stats.totalLong")}
       >
         {hasRoomForFullText
-          ? t("pokedex.stats.totalLong")
-          : t("pokedex.stats.total")}
+          ? t("pokedex.stats.totalLong")+ " / EVs"
+          : t("pokedex.stats.total") + " / EVs"}
```

Comment:
The term "EVs" would need to be translated... https://bulbapedia.bulbagarden.net/wiki/Effort_values#In_other_languages